lib/device_api/android.rb

```diff
@@ -28,7 +28,7 @@ def self.device(serial)
         raise DeviceAPI::BadSerialString.new("serial was '#{serial.nil? ? 'nil' : serial}'")
       end
       state = ADB.get_state(serial)
-      DeviceAPI::Android::Device.create( self.get_device_type(serial),  { serial: serial, state: state })
+      DeviceAPI::Android::Device.create( self.get_device_type({serial: state}),  { serial: serial, state: state })
```

Comment:
Line is too long. [114/80]<br>Redundant curly braces around a hash parameter.<br>Unnecessary spacing detected.<br>Redundant `self` detected.<br>Space inside { missing.<br>Space inside } missing.<br>Space inside parentheses detected.